<launch>
  <!-- argument list -->
  <arg name="camera_id" default="front_camera"/>
  <arg name="image_src" default="/image/raw" />
  <arg name="pretrained_model_file" default="$(find region_tlr_effnet)/data/model_best_lib.pt" />
  <arg name="use_gpu" default="true" />


  <!-- run region_tlr_effNet node -->
  <node pkg="region_tlr_effnet" name="region_tlr_effnet" type="region_tlr_effnet" output="log">
    <param name="image_raw_topic" type="str" value="$(arg camera_id)$(arg image_src)" />
    <param name="pretrained_model_file" type="str" value="$(arg pretrained_model_file)" />
    <param name="use_gpu" type="bool" value="$(arg use_gpu)" />
  </node>
</launch>
